tests/acceptance/features/login.feature

```diff
@@ -9,7 +9,9 @@ Feature: login users
 
 	Scenario: admin login
 		Given the user has browsed to the login page
-		When the user logs in with username "admin" and password "admin" using the webUI
+		When the user clicks the authenticate button
```

Comment:
this is from #307 - please rebase on that PR and let this PR merge into #307 - THX